IBAnimatable/FillDesignable.swift

```diff
@@ -34,6 +37,62 @@ public extension FillDesignable where Self: UITableViewCell {
     if let unwrappedFillColor = fillColor {
       backgroundColor = unwrappedFillColor
       contentView.backgroundColor = unwrappedFillColor
+    } else if let unwrappedPredefinedColor = predefinedColorFromString(predefinedColor) {
+      backgroundColor = unwrappedPredefinedColor
+      contentView.backgroundColor = unwrappedPredefinedColor
     }
   }
 }
+
+private extension FillDesignable {
+  
+  func predefinedColorFromString(predefinedColor: String?) -> UIColor? {
+    guard let unwrappedColorTypeString = predefinedColor, colorType = ColorType(rawValue: unwrappedColorTypeString) else {
+      return nil
+    }
+    
+    switch colorType {
+    case .Emerland:
```

Comment:
I understand your point. I like the idea to have the completion when tapping .Flat. I just updated the PR. For the name, it's predefined colors, so I don't think it's a big issue to have longer name.